packages/rln/karma.conf.cjs

```diff
@@ -47,14 +47,15 @@ module.exports = function (config) {
 
     client: {
       mocha: {
-        timeout: 180000 // 3 minutes
```

Comment:
what caused this increase?

Reply:
it's an increase in timeout; new tests needed more time (more than previously defined timeout)

Reply:
but why? I don't see tests being changed that much

Reply:
some tests timed out with the older limit -- if youre interested, i can share which tests needed more time